scripts/GuestList.js

```diff
@@ -0,0 +1,31 @@
+//import { getGuests } from "./database.js"
+//?import { getLocations } from "./database.js"
+
+//? locations = getLocations()
+
+//guests = getGuests()
+
+
+//Meant to iterate through to make the guest list that will display on the right side of the website.
+// export const guestList = () => {
+    // let guestHTML = "<ul>"
+    
+    // for (const guest of guests) {
+    //     guestHTML += `<li>${guest.name}</li>`
+    // }
+    // guestHTML += "</ul>"
+
+    // return guestHTML
+// }
+
+
+//?Potentially used for the pairing of guest to there location maybe
```

Comment:
I think the requirements are just to have the guest names displayed, so I don't think we need anything below this line

Reply:
All the code that was irrelevant has been removed and should be better for this module.